### DISCUSSION TITLE
redesign-blog

### DISCUSSION THREAD

**Discussion by giscus** [mrocklin/mrocklin.github.io#4] [Giscus]
# redesign-blog

How we read and write shapes how we think Twitter chaos made us rethink our engagement on the web. Personally, I felt smarter and happier pre-Twitter. During this period I engaged by reading and wr...

https://matthewrocklin.com/redesign-blog.html

**Comment by rabernat** [comment]
This is great Matt! Love the new design!

One piece of feedback. There are now no dates to be found anywhere on your blog. I think putting the publication date somewhere on the post (even if not in the URL) is very important context for readers.

**Reply by mrocklin** [reply]
Sure.  Done.  @asmeurer was also asking for this.

**Reply by mrocklin** [reply]
I am curious though, what is your motivation for the date?  What value does this sort of context provide do you think?  

My sense is that at least for evergreen content the date is not relevant (we don't often put dates on individual documentation pages for example)

**Reply by rabernat** [reply]
To use this blog post as an example...Twitter is in a state of churn right now. When reading about your decision to shift away from Twitter, it's useful for me to know you're wrote this in January 2023, as opposed to January 2022.

**Reply by rsignell-usgs** [reply]
Matt, a big plus one on the value of your evergreen blog posts!  The reason I'm here is because I was searching for your post on crafting minimal bug reports, which I remember as especially impactful for me.   I wanted to share it with some early career folks I'm mentoring.  So I'm really grateful you are at this again.  Thank you for taking the extra time to do this, especially with everything else you've got going on!

**Reply by mrocklin** [reply]
Thanks Rich!
